psignifit/likelihood.py

```diff
@@ -5,7 +5,7 @@
 import scipy.special as sp
 from scipy import optimize
 
-from .utils import fp_error_handler, PsignifitException
+from .utils import fp_error_handler, PsignifitException, integral_weights
```

Comment:
the function `integral_weights` should be moved to the `likelihood.py` file, if nothing else uses it.